### PR TITLE
Change "authentication token" to "API token" in InfluxDB 2 docs

### LINKED ISSUE #2436: Update "authentication tokens" to "API tokens"
@russorat has proposed that we update the term "authentication token" to "API token" throughout the 2.0 docs.

I'm fine with this change, but it will need to be done carefully. There may be references to "authentication tokens" for third parties for which the term is appropriate.

Does anyone have any concerns about this change?

### PR DESCRIPTION
**Pull request by pierwill**
Closes #2436

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/install.md [hunk 1]
```diff
@@ -585,7 +585,7 @@ to collect and send data to:
 ## Set up InfluxDB
 
 The initial setup process for InfluxDB walks through creating a default organization,
-user, bucket, and Admin authentication token.
+user, bucket, and Admin API token.
```
Comment (sanderson) [inline content/influxdb/v2.0/install.md:588]:
```suggestion
user, bucket, and Operator API token.
```

File: content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ To send notifications about changes in your data, start by creating a notificati
 5.  In the **Name** and **Description** fields, enter a name and description for the endpoint.
 6.  Enter information to connect to the endpoint:
 
-    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the authentication token in the **Token** field.
+    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the API token in the **Token** field.
```
Comment (sanderson) [inline content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md:29]: This one is referring to the token that gets sent with the `Authorization` header. We can probably just remove authentication/API.

```suggestion
    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the token in the **Token** field.
```

File: content/influxdb/v2.0/reference/config-options.md [hunk 3]
```diff
@@ -3013,7 +3013,7 @@ vault-tls-server-name = "secure.example.com"
 ---
 
 ### vault-token
-Specifies the Vault authentication token use when authenticating with Vault.
+Specifies the Vault API token use when authenticating with Vault.
```
Comment (sanderson) [inline content/influxdb/v2.0/reference/config-options.md:3016]: We can just remove `API` here.

```suggestion
Specifies the Vault token use when authenticating with Vault.
```

**Review by sanderson — APPROVED** [review]
Some minor suggestions. Thanks for doing this. Let's be sure to coordinate merging this PR with the PMs in case there are any unintended consequences elsewhere.

File: content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md [hunk 4]
```diff
@@ -26,7 +26,7 @@ To send notifications about changes in your data, start by creating a notificati
 5.  In the **Name** and **Description** fields, enter a name and description for the endpoint.
 6.  Enter information to connect to the endpoint:
 
-    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the authentication token in the **Token** field.
+    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the token in the **Token** field.
```
Comment (russorat) [inline content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md:29]:
```suggestion
    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with an API token, select **Bearer**, and then enter the API token in the **Token** field.
```

File: content/influxdb/v2.0/reference/cli/influx/export/all.md [hunk 5]
```diff
@@ -22,7 +22,7 @@ _For detailed examples of exporting InfluxDB templates, see
 {{% note %}}
 To export resources as a template, you must use the **Operator token** created for
 the initial InfluxDB user or an **All-Access token**.
-For information about creating an All-Access token, see [Create an authentication token](/influxdb/v2.0/security/tokens/create-token/).
+For information about creating an All-Access token, see [Create an API token](/influxdb/v2.0/security/tokens/create-token/).
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/export/all.md:25]:
```suggestion
For information about creating an All-Access API token, see [Create an API token](/influxdb/v2.0/security/tokens/create-token/).
```

File: content/influxdb/v2.0/reference/cli/influx/export/stack.md [hunk 6]
```diff
@@ -19,7 +19,7 @@ All `metadata.name` fields remain the same.
 {{% note %}}
 To export resources as a template, you must use the **Operator token** created for
 the initial InfluxDB user or an **All-Access token**.
-For information about creating an All-Access token, see [Create an authentication token](/influxdb/v2.0/security/tokens/create-token/).
+For information about creating an All-Access token, see [Create an API token](/influxdb/v2.0/security/tokens/create-token/).
```
Comment (russorat) [inline content/influxdb/v2.0/reference/cli/influx/export/stack.md:22]:
```suggestion
For information about creating an All-Access API token, see [Create an API token](/influxdb/v2.0/security/tokens/create-token/).
```

File: content/influxdb/v2.0/security/tokens/_index.md [hunk 7]
```diff
@@ -12,12 +12,12 @@ menu:
 weight: 103
 ---
 
-InfluxDB **authentication tokens** ensure secure interaction between users and data.
+InfluxDB **API tokens** ensure secure interaction between users and data.
 A token belongs to an organization and identifies InfluxDB permissions within the organization.
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/_index.md:16]:
```suggestion
An API token belongs to an organization and identifies InfluxDB permissions within the organization.
```

File: content/influxdb/v2.0/security/tokens/_index.md [hunk 8]
```diff
@@ -12,12 +12,12 @@ menu:
 weight: 103
 ---
 
-InfluxDB **authentication tokens** ensure secure interaction between users and data.
+InfluxDB **API tokens** ensure secure interaction between users and data.
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/_index.md:15]:
```suggestion
InfluxDB **API tokens** ensure secure interaction between InfluxDB and external tools such as clients or applications.
```

File: content/influxdb/v2.0/security/tokens/_index.md [hunk 9]
```diff
@@ -12,12 +12,12 @@ menu:
...
 A token belongs to an organization and identifies InfluxDB permissions within the organization.
 
-Learn how to create, view, update, or delete an authentication token.
+Learn how to create, view, update, or delete an API token.
 
-## Authentication token types
+## API token types
 
 - [Operator token](#operator-token)
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/_index.md:22]:
```suggestion
- [Operator API token](#operator-token)
```

File: content/influxdb/v2.0/security/tokens/_index.md [hunk 10]
```diff
@@ -12,12 +12,12 @@ menu:
...
 - [All-Access token](#all-access-token)
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/_index.md:23]:
```suggestion
- [All-Access API token](#all-access-token)
```

File: content/influxdb/v2.0/security/tokens/create-token.md [hunk 11]
```diff
@@ -11,7 +11,7 @@ menu:
 weight: 201
 ---
 
-Create authentication tokens using the InfluxDB user interface (UI) or the `influx`
+Create API tokens using the InfluxDB user interface (UI) or the `influx`
 command line interface (CLI).
 
 Tokens are visible only to the user who created them and stop working when the user is deactivated. We recommend creating a generic IT user to create and manage tokens for writing data.
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/create-token.md:17]:
```suggestion
API Tokens are visible only to the user who created them and stop working when the user is deactivated. We recommend creating a generic IT user to create and manage tokens for writing data.
```

File: content/influxdb/v2.0/security/tokens/delete-token.md [hunk 12]
```diff
@@ -11,7 +11,7 @@ menu:
 weight: 204
 ---
 
-Delete authentication tokens from the InfluxDB user interface (UI) or the `influx` command line interface (CLI).
+Delete API tokens from the InfluxDB user interface (UI) or the `influx` command line interface (CLI).
 Once deleted, all users and external integrations using the token will no longer
```
Comment (russorat) [inline content/influxdb/v2.0/security/tokens/delete-token.md:15]:
```suggestion
Once deleted, all users and external integrations using the API token will no longer
```

File: content/influxdb/v2.0/write-data/developer-tools/api.md [hunk 13]
```diff
@@ -16,7 +16,7 @@ Use the `POST` request method and include the following in your request:
 | Organization         | Use the `org` query parameter in your request URL.       |
 | Bucket               | Use the `bucket` query parameter in your request URL.    |
 | Precision            | Use the `precision` query parameter in your request URL. |
-| Authentication token | Use the `Authorization: Token` header.                   |
+| API token | Use the `Authorization: Token` header.                   |
```
Comment (russorat) [inline content/influxdb/v2.0/write-data/developer-tools/api.md:19]:
```suggestion
| API token | Use the `Authorization: Token <API token>` header.                   |
```

File: content/influxdb/v2.0/write-data/load-data.md [hunk 14]
```diff
@@ -24,7 +24,7 @@ Load data from the following sources in the InfluxDB user interface (UI):
    - Enter a specific client library to search for in the **Search data writing methods** field.
    - Scroll down to browse available client libraries.
 3. Click the client library to load data from.
-4. Under **Code Sample Options**, you'll see a list of your InfluxDB [tokens](/influxdb/v2.0/reference/glossary/#token) and [buckets](/influxdb/v2.0/reference/glossary/#bucket). Select both an authentication token and a bucket to write your data to. The selected token and bucket are automatically added to scripts on the page that you can use to initialize a client and write data.
+4. Under **Code Sample Options**, you'll see a list of your InfluxDB [tokens](/influxdb/v2.0/reference/glossary/#token) and [buckets](/influxdb/v2.0/reference/glossary/#bucket). Select both an API token and a bucket to write your data to. The selected token and bucket are automatically added to scripts on the page that you can use to initialize a client and write data.
```
Comment (russorat) [inline content/influxdb/v2.0/write-data/load-data.md:27]:
```suggestion
4. Under **Code Sample Options**, you'll see a list of your InfluxDB [tokens](/influxdb/v2.0/reference/glossary/#token) and [buckets](/influxdb/v2.0/reference/glossary/#bucket). Select both an API token and a bucket to write your data to. The selected API token and bucket are automatically added to scripts on the page that you can use to initialize a client and write data.
```

File: content/influxdb/v2.0/write-data/no-code/use-telegraf/auto-config.md [hunk 15]
```diff
@@ -81,12 +81,12 @@ If you plan to monitor a Windows host using the System plugin, you must complete
...
-Requests to the [InfluxDB v2 API](/influxdb/v2.0/reference/api/) must include an authentication token.
+Requests to the [InfluxDB v2 API](/influxdb/v2.0/reference/api/) must include an API token.
 A token identifies specific permissions to the InfluxDB instance.
 
 ### Configure your token as an environment variable
 
-1. Find your authentication token. _For information about viewing tokens, see [View tokens](/influxdb/v2.0/security/tokens/view-tokens/)._
+1. Find your API token. _For information about viewing tokens, see [View tokens](/influxdb/v2.0/security/tokens/view-tokens/)._
 
 2. To configure your token as the `INFLUX_TOKEN` environment variable, run the command appropriate for your operating system and command-line tool:
```
Comment (russorat) [inline content/influxdb/v2.0/write-data/no-code/use-telegraf/auto-config.md:91]:
```suggestion
2. To configure your API token as the `INFLUX_TOKEN` environment variable, run the command appropriate for your operating system and command-line tool:
```

**Review by sanderson — APPROVED** [review]

**Review by kelseiv — APPROVED** [review]
Did we intentionally leave the Cloud docs out for this go-round?

**Comment by pierwill** [comment]
> Did we intentionally leave the Cloud docs out for this go-round?

Good catch! It was unintentional...

**Comment by pierwill** [comment]
Conflicts resolved, and Cloud terminology updated.

**Review by kelseiv — APPROVED** [review]

**Comment by pierwill** [comment]
Once https://github.com/influxdata/ui/pull/2200 closes, let's merge this.

**Pull request merged**